ecosystem/indexer-grpc/indexer-grpc-data-service/src/metrics.rs

```diff
@@ -15,7 +15,13 @@ pub static LATEST_PROCESSED_VERSION_PER_PROCESSOR: Lazy<IntGaugeVec> = Lazy::new
     register_int_gauge_vec!(
         "indexer_grpc_data_service_with_user_latest_processed_version",
```

Comment:
we might need to rename the metrics to `_v2` so it does not conflict with existing metrics

Reply:
This just adds new labels to the metrics, all the existing queries should still work.